inspirehep/search/facets.py

```diff
@@ -98,3 +98,32 @@ def citation_summary():
             }
         },
     }
+
+
+def citations_by_year():
+    excluded_filters = [
+        "citeable",
+        "refereed",
+        "citation_count",
+        "author_count",
+        "earliest_date",
+        "collaboration",
+    ]
```

Comment:
why?

Reply:
It is the most reasonable setup in my opinion. Do not enable things we don't need. For now this will be only used to retrieve authors by facet names. When this requirement will change this also can be easily changed.

Reply:
I have the opposite opinion: don't write *code* you don't need. But it's your call.